.github/workflows/manual.yml

```diff
@@ -1,7 +1,13 @@
 # This is a basic workflow that is manually triggered
-
 name: Manual workflow
 
+# Can be run automatically with the following:
+# on:
+#   pull_request:
```

Comment:
Why is this commented ?

Reply:
OSX doesn't function as expected. Linux does, so leaving it to be run manually for now, not with every commit.